src/java/org/httpkit/HttpUtils.java

```diff
@@ -257,6 +257,14 @@ public static String getHost(URI uri) {
         }
         return host;
     }
+    
+    public static String getProxyHost(URI uri){
```

Comment:
I like that because it's closer to clj-http I think but we would also have to put in :proxy-scheme (default to http?) might get verbose from a config perspective. thoughts?